File: Week-02/day-04/matchmaking.py
# Join the two lists by matching one girl with one boy in the order list
# Exepected output: ["Eve", "Joe", "Ashley", "Fred"...]

girls = ["Eve","Ashley","Bözsi","Kat","Jane"]
boys = ["Joe","Fred","Béla","Todd","Neef","Jeff"]
order = []
n = max(len(girls), len(boys))

for nr in range(0, n):
    if nr <= len(girls)-1:
        order.append(girls[nr])
    else:
        pass
    if nr <= len(boys)-1:
        order.append(boys[nr])
    else:
        pass
print(order)

File: Week-07/Day-03/matchmaking.js
'use strict';
// Join the two array by matching one girl with one boy in the order array
// Exepected output: ["Eve", "Joe", "Ashley", "Fred"...]

var girls = ["Eve","Ashley","Bözsi","Kat","Jane"];
var boys = ["Joe","Fred","Béla","Todd","Neef","Jeff"];
var order = [];



var minLength = Math.min(girls.length, boys.length)

for (var i = 0; i < minLength; i++) {
  order.push(girls[i]);
  order.push(boys[i]);
}

console.log(order);
